src/main/java/org/apache/sling/discovery/base/connectors/ping/TopologyConnectorClient.java

```diff
@@ -112,9 +112,9 @@ public class TopologyConnectorClient implements
     /** SLING-3382: unix-time at which point the backoff-period ends and pings can be sent again **/
     private long backoffPeriodEnd = -1;
     
-    TopologyConnectorClient(final ClusterViewService clusterViewService,
```

Comment:
if this has always been package protected I guess there is no need to export this either.

Reply:
+1, this does not need exporting